Source/WebKit/NetworkProcess/DatabaseUtilities.cpp

```diff
@@ -328,4 +328,37 @@ void DatabaseUtilities::migrateDataToNewTablesIfNecessary()
     transaction.commit();
 }
 
+Vector<String> DatabaseUtilities::columnsForTable(ASCIILiteral tableName)
+{
+    auto statement = m_database.prepareStatementSlow(makeString("PRAGMA table_info(", tableName, ")"));
+
+    if (!statement) {
+        RELEASE_LOG_ERROR(PrivateClickMeasurement, "%p - Database::columnsForTable Unable to prepare statement to fetch schema for table, error message: %" PRIVATE_LOG_STRING, this, m_database.lastErrorMsg());
```

Comment:
I see you're using the PrivateClickMeasurement channel/category here. Copy pasta?

Reply:
Or maybe this is infrastructure that used to be only for PCM but is also used for ITP?

Reply:
That's a problem that already existed in this file.  It is code that was originally written for PCM and ITP shared, then those two were separated, and this is the location of shared database utility code.  I keep the PrivateClickMeasurement log channel name because it is half correct.